### PR TITLE
k8s common resource labels

### PR DESCRIPTION
**Pull request by AdiZav**
##### Short description:
Common k8s resource labes
##### More details:
Something to consider if we end up adding functionality like adding/getting/removing resource labels -
Most resources have only some of these labels if any at all

### CODE REVIEW

File: ocp_resources/resource.py [hunk 1]
```diff
@@ -223,6 +223,15 @@ class Resource(object):
     api_version = None
     singular_name = None
 
+    class Label:
+        NAME = "app.kubernetes.io/name"
```
Comment (myakove) [inline ocp_resources/resource.py:227]: I think what @rnetser wanted is to have
`app.kubernetes.io` under `class ApiGroup`

Comment (rnetser) [reply ocp_resources/resource.py:227]: Yes (i.e add "app.kubernetes.io" under  ApiGroup and add Label.NAME)

Comment (AdiZav) [reply ocp_resources/resource.py:227]: So when used in cnv-tests it should look like:
`    preallocation_requested_annotation = (
        f"{NamespacedResource.ApiGroup.CDI_KUBEVIRT_IO}/{preallocation_requested}"
    )`?

**Review by myakove — CHANGES_REQUESTED** [review]
Set in class ApiGroup

File: ocp_resources/resource.py [hunk 2]
```diff
@@ -223,6 +223,15 @@ class Resource(object):
     api_version = None
     singular_name = None
 
+    class Label:
```
Comment (myakove) [inline ocp_resources/resource.py:226]: class label can be removed.

**Review by myakove — CHANGES_REQUESTED** [review]

**Pull request merged**